src/main/java/dev/openfeature/sdk/EventSupport.java

```diff
@@ -122,11 +121,11 @@ public void removeGlobalHandler(ProviderEvent event, Consumer<EventDetails> hand
     }
 
     /**
-     * Get all client names for which we have event handlers registered.
+     * Get all domain names for which we have event handlers registered.
      * 
-     * @return set of client names
+     * @return set of domain names
      */
-    public Set<String> getAllClientNames() {
+    public Set<String> getAllDomainNames() {
```

Comment:
I think we can avoid this change, and even make `getAllClientNames` package private 🤔